packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -6012,6 +6013,34 @@ describe('plugin-meetings', () => {
       });
     });
 
+    describe('#setSendNamedMediaGroup', () => {
+      beforeEach(() => {
+        meeting.sendSlotManager.setNamedMediaGroups = sinon.stub().returns(undefined);
+      });
+      it('should throw error if not audio type', () => {
+        // try {
```

Comment:
commented out code - if not needed, remove it, if needed, uncomment it :-)

Reply:
Removed